dashboard/db/migrate/20201201202751_add_new_columns_to_schools.rb

```diff
@@ -0,0 +1,9 @@
+class AddNewColumnsToSchools < ActiveRecord::Migration[5.2]
+  def change
+    add_column :schools, :school_category, :string
+    add_column :schools, :last_known_school_year_open, :string, limit: 9
+    add_index :schools, :last_known_school_year_open
+
+    add_column :school_districts, :last_known_school_year_open, :string, limit: 9, after: :zip
```

Comment:
(for my information) what does after: :zip do?

Reply:
Places the new column after the `zip` column (ie, before `updated_at` and `created_at`), instead of at the end (default). I kind of like having those timestamp columns as the last two columns in a model, but that rule's already been violated for `schools` so I didn't bother there :).